packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -89,6 +90,7 @@ const NotificationDetail: React.FC = () => {
   );
 
   const isCancelled = useIsCancelled({ notification });
+  const isViewed = notification.notificationStatus === NotificationStatus.VIEWED;
```

Comment:
This is not totally correct. We have to check if notification has been viewed or not. This check must be valid for all those status after viewed. With this check we are checking that the "current" status is Viewed but we are not considering other statuses after that.

TIPS: check how was done in the pg app